src/Ethereum.sol

```diff
@@ -0,0 +1,111 @@
+    address internal constant WBTC_ORACLE            = 0x230E0321Cf38F09e247e50Afc7801EA2351fe56F;
+    address internal constant WBTC_STABLE_DEBT_TOKEN = 0x4b29e6cBeE62935CfC92efcB3839eD2c2F35C1d9;
+    address internal constant WBTC_DEBT_TOKEN        = 0xf6fEe3A8aC8040C3d6d81d9A4a168516Ec9B51D2;
+
+    address internal constant WETH_ATOKEN            = 0x59cD1C87501baa753d0B5B5Ab5D8416A45cD71DB;
+    address internal constant WETH_IRM               = 0xD7A8461e6aF708a086D8285f8fD900309336347c;
+    address internal constant WETH_ORACLE            = 0x5f4eC3Df9cbd43714FE2740f5E3616155c5b8419;
+    address internal constant WETH_STABLE_DEBT_TOKEN = 0x3c6b93D38ffA15ea995D1BC950d5D0Fa6b22bD05;
+    address internal constant WETH_DEBT_TOKEN        = 0x2e7576042566f8D6990e07A1B61Ad1efd86Ae70d;
+
+    address internal constant WSTETH_ATOKEN            = 0x12B54025C112Aa61fAce2CDB7118740875A566E9;
+    address internal constant WSTETH_IRM               = 0x0D56700c90a690D8795D6C148aCD94b12932f4E3;
+    address internal constant WSTETH_ORACLE            = 0xA9F30e6ED4098e9439B2ac8aEA2d3fc26BcEbb45;
+    address internal constant WSTETH_STABLE_DEBT_TOKEN = 0x9832D969a0c8662D98fFf334A4ba7FeE62b109C2;
+    address internal constant WSTETH_DEBT_TOKEN        = 0xd5c3E3B566a42A6110513Ac7670C1a86D76E13E6;
+
+    /******************************************************************************************************************/
+    /*** Data Provider Addresses                                                                                    ***/
+    /******************************************************************************************************************/
+
+    address internal constant PROTOCOL_DATA_PROVIDER     = 0xFc21d6d146E6086B8359705C8b28512a983db0cb;
+    address internal constant UI_INCENTIVE_DATA_PROVIDER = 0xA7F8A757C4f7696c015B595F51B2901AC0121B18;
+    address internal constant UI_POOL_DATA_PROVIDER      = 0xF028c2F4b19898718fD0F77b9b881CbfdAa5e8Bb;
+    address internal constant WALLET_BALANCE_PROVIDER    = 0xd2AeF86F51F92E8e49F42454c287AE4879D1BeDc;
+
+    /******************************************************************************************************************/
+    /*** Implementation Addresses                                                                                   ***/
+    /******************************************************************************************************************/
+
+    address internal constant A_TOKEN_IMPL             = 0x6175ddEc3B9b38c88157C10A01ed4A3fa8639cC6;
+    address internal constant DAI_TREASURY_IMPL        = 0xF1E57711Eb5F897b415de1aEFCB64d9BAe58D312;  // Deprecated?
+    address internal constant INCENTIVES_IMPL          = 0x0ee554F6A1f7a4Cb4f82D4C124DdC2AD3E37fde1;
+    address internal constant POOL_CONFIGURATOR_IMPL   = 0xF7b656C95420194b79687fc86D965FB51DA4799F;
+    address internal constant POOL_IMPL                = 0x8115366Ca7Cf280a760f0bC0F6Db3026e2437115;
```

Comment:
We updated this to `0xb40f6d584081ac2b0fd84c846dba3c1417889304`.

Reply:
Updated